cmd/rook/operator.go

```diff
@@ -37,6 +39,8 @@ https://github.com/rook/rook`,
 }
 
 func init() {
+	operatorCmd.Flags().DurationVar(&mon.HealthCheckInterval, "mon-healthcheck-interval", 20*time.Second, "mon health check interval (duration)")
```

Comment:
can these defaults reference the the mon package defaults?

Reply:
Like this?